README.md

```diff
@@ -135,6 +135,49 @@ AnyCable, k6, WebSockets, and Yabeda](https://evilmartians.com/chronicles/real-t
 - [k6 Extensions](https://grafana.com/docs/k6/latest/extensions/)
 - [GitHub Topic: xk6](https://github.com/topics/xk6) - Explore k6 extensions tagged with the xk6 label.
 
+### Official
+
+- [xk6-client-tracing](https://github.com/grafana/xk6-client-tracing) - Client for load testing distributed tracing backends.
+- [xk6-dashboard](https://github.com/grafana/xk6-dashboard) - Create a web-based metrics dashboard.
+- [xk6-disruptor](https://github.com/grafana/xk6-disruptor) - Inject faults to test 💣.
+- [xk6-exec](https://github.com/grafana/xk6-exec) - Run external commands.
+- [xk6-kubernetes](https://github.com/grafana/xk6-kubernetes) - Interact with Kubernetes clusters.
+- [xk6-loki](https://github.com/grafana/xk6-loki) - Client for load testing Loki.
+- [xk6-notification](https://github.com/grafana/xk6-notification) - Create notifications.
+- [xk6-output-influxdb](https://github.com/grafana/xk6-output-influxdb) - Export results to InfluxDB v2.
+- [xk6-output-kafka](https://github.com/grafana/xk6-output-kafka) - Export k6 results in real-time to Kafka.
+- [xk6-output-timescaledb](https://github.com/grafana/xk6-output-timescaledb) - Export k6 results to TimescaleDB.
+- [xk6-sql](https://github.com/grafana/xk6-sql) - Load-test SQL Servers (PostgreSQL, MySQL and SQLite3 for now).
+- [xk6-ssh](https://github.com/grafana/xk6-ssh) - SSH.
+
+### Community
+
+- [xk6-cable](https://github.com/anycable/xk6-cable) - Test Action Cable and AnyCable functionality.
+- [xk6-client-prometheus-remote](https://github.com/grafana/xk6-client-prometheus-remote) - Test Prometheus Remote Write performance.
```

Comment:
I believe this extension can be considered officially supported.

Reply:
OK, I'll move it to the official section. Could you please open a PR in the k6-docs repo by setting the official flag to true for this extension.

Reply:
Sure, will do!

Reply:
https://github.com/grafana/k6-docs/pull/1679

Reply:
The above PR is merged now.

Reply:
Done, I moved to official section